docs/elementExtensions.md

```diff
@@ -0,0 +1,73 @@
+# Element extensions developer guide
```

Comment:
Add to:

https://github.com/PrairieLearn/PrairieLearn/blob/2260c3d99f0cb73f01463228b9a10f65b71804c2/mkdocs.yml#L37